self-tests.yml

```diff
@@ -132,7 +134,7 @@ variables:
 
             cd gopath/src/github.com/evergreen-ci/evergreen
             mkdir -p clients
-            echo "foo" >| clients/version
+            echo "$(git rev-parse HEAD)" >| clients/version
```

Comment:
I think you can omit this line entirely. The clients/version file isn't used anymore and we just use `evergreen.BuildRevision`

Reply:
done